src/lib/support/jsontlv/TlvToJson.h

```diff
@@ -48,4 +48,9 @@ std::string PrettyPrintJsonString(const std::string & jsonString);
  */
 std::string MakeJsonSingleLine(const std::string & jsonString);
 
+/*
+ * Given a TLV encoded byte array, this function add the outer container, encode the tag Id and converts it into JSON object.
```

Comment:
What "the outer container"

What "tag id"?

This API needs a much clearer explanation of what it does and when you might want to use it.  And why this is an API everyone would want to use, as opposed to something used by one specific callsite.

Reply:
updated, thanks